jobs/issues_status.rb

```diff
@@ -76,4 +108,22 @@
              trend_class: trend_class_closed,
              arrow: 'icon-arrow-' + trend_class_closed
             )
+
+  # ----------- PULL REQUEST STATS -------------------------- #
+
+  # rubocop:disable Style/RescueModifier
```

Comment:
I disabled this particular cop a while back because Hound was complaining about it, but this is existing code from the forked repo, and fixing it to please Rubocop/Hound should be a separate task IMO. I didn't write this code from scratch in this PR. It was copied and pasted from `pull_requests.rb` that came from the forked repo and that I deleted in this PR. 

If Hound had complained about new code I wrote, I would definitely fix it, but in this case, I didn't think it was worth the time to fix it. Since this project is not billable, I try to prioritize the work that achieves our goal.